adr/0011-python-application-deployment.md

```diff
@@ -0,0 +1,75 @@
+# Python application building and deployment
```

Comment:
`... via containers` (as the title has to capture the decision in ~50 characters)